### ISSUE TITLE
Atom closing unexpectedly

### ISSUE THREAD

**Issue by StuffnThingz** [Media-Ed-Online/intro-web-dev#39]
Anyone had an experience with Atom just closing down while you're working?

**Comment by JohnYingling** [comment]
Yep.  Crashes often on my Mac.  Strange.  I have this weird feeling I didn't install something right, as on my second time trying to install, the previous version gives me a white box that simply crashes.  Now I'm too far in I don't want to re-do stuff or blow anything up....

**Comment by StuffnThingz** [comment]
Weird.  I'm on a PC.  It's only done it a few times, but it's annoying.

**Comment by CMHauge** [comment]
Mine does that as well, but only when saving. (I save a lot, purely so I don't lose my progress). My best guess is that it doesn't like having to save and update the preview tab at the same time. I still have to fiddle with it and see if I can pinpoint it, but that's my theory so far!

**Comment by JustineEvansUM** [comment]
I'm curious if this is happening more for Mac or PC users. Chime in, please!

@JonSwallow started a great post similar to this. [Check it out](https://github.com/Media-Ed-Online/intro-web-dev/issues/36), see if it helps!

**Comment by FriedmanM** [comment]
I still cannot use my atom-html-preview while editing an html file or else it will crash atom. I am on a Mac on MacOS 10.13 High Sierra (GM Beta). I recommend people on Macs to try disabling that add-on and see if it still crashes. I remedy this by having the file open in my browser and refreshing the page after saving to see the changes.

**Comment by CMHauge** [comment]
I'm on PC, Windows 10, and Atom crashed on me a couple dozen times the other day, which is especially frustrating when you're already sick and fuzzy-brained. (Food poisoning, yay.)

Anyway, I thought originally the crashing was just happening when I was saving, but I actually had it crash a couple times the other day when I was typing. I also found that the html preview seems to cause more crashes than anything else. Opening the html file in your browser definitely seems to be a good work around!

**Comment by JustineEvansUM** [comment]
@CMHauge This is a great workaround. It seems that the HTML preview has been troublesome for many, and I've seen many PC users in my office over the last few weeks. I'm combing the boards, but I can't quite find what the prevailing issue is.

**PC USERS:** If you're having crash issues, try disabling your HTML preview package(s), and use your browser to preview your `.html` files. Just refresh after every change you make in Atom!

**Comment by camkincaid** [comment]
I am glad I found this thread! I am using a mac on the latest iOS and every time I try to HTML preview it shows me a white window and then crashes. Help!!! Any solutions found? @JustineEvansUM

**Comment by JustineEvansUM** [comment]
Hey, team;

My Atom closes with HTML preview, so I'm officially calling that the problem. Who knew!
